src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -570,7 +568,7 @@ class MediaBaseViewer extends BaseViewer {
      * @param {number} start - start time in seconds
      * @param {number} end - end time in seconds
      * @emits play
-     * @return {void}
+     * @return {Promise}
```

Comment:
This is misleading because this method can return undefined as a result of `this.mediaEl.play` in older browsers. Could we make it consistently return a promise? Likely that would mean moving the check to see if `this.mediaEl.play` returns a promise in here. @jstoffan had a suggestion to throw an error if it's not a promise that `autoplay` can catch and handle to do the fallback logic of `this.mediaEl.autoplay = true;`

Would probably need to differentiate this situation vs the `NotAllowedError` due to not muting or user interaction in the `catch` callback in `autoplay`